xikolo-ios/RichtextViewController.swift

```diff
@@ -14,7 +14,7 @@ class RichtextViewController: AbstractItemRichtextViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        titleLabel.text = courseItem?.title ?? "default title"
+        titleLabel.text = courseItem?.title ?? ""
```

Comment:
Why would it? You can set a `UILabel`'s text to `nil` without a problem. It will then default to the empty string by itself.